spec/javascripts/app/views/notifications_view_spec.js

```diff
@@ -1,36 +1,17 @@
-describe("app.views.Notifications", function(){
+describe("app.views.Notifications", function() {
+  beforeEach(function() {
+    this.collection = new app.collections.Notifications();
+    this.collection.fetch();
+    jasmine.Ajax.requests.mostRecent().respondWith({
+      status: 200,
+      responseText: spec.readFixture("notifications_collection")
+    });
+  });
+
   context("on the notifications page", function() {
     beforeEach(function() {
       spec.loadFixture("notifications");
-      this.view = new app.views.Notifications({el: "#notifications_container"});
-    });
-
-    describe("mark read", function() {
-      beforeEach(function() {
-        this.unreadN = $(".stream-element.unread").first();
-        this.guid = this.unreadN.data("guid");
-      });
-
-      it("calls 'setRead'", function() {
-        spyOn(this.view, "setRead");
-        this.unreadN.find(".unread-toggle").trigger("click");
-
-        expect(this.view.setRead).toHaveBeenCalledWith(this.guid);
-      });
-    });
-
-    describe("mark unread", function() {
-      beforeEach(function() {
-        this.readN = $(".stream-element.read").first();
-        this.guid = this.readN.data("guid");
-      });
-
-      it("calls 'setUnread'", function() {
-        spyOn(this.view, "setUnread");
-        this.readN.find(".unread-toggle").trigger("click");
-
-        expect(this.view.setUnread).toHaveBeenCalledWith(this.guid);
-      });
```

Comment:
Am I missing something or are there now no tests for marking notifications as read/unread via click?